sos/plugins/origin.py

```diff
@@ -45,14 +45,21 @@ class OpenShiftOrigin(Plugin):
     files = None  # file lists assigned after path setup below
     profiles = ('openshift',)
 
+    option_list = [
+        ("diag", "run 'oc adm diagnostics' to collect its output",
+         'fast', True),
+        ("diag-prevent", "set --prevent-modifications on 'oc adm diagnostics'",
+         'fast', False),
```

Comment:
Wouldn't we actually want this `True` by default? Are there any harmful modifications possible is this option is absent?

Reply:
There was a discussion about this in [bug 1463509](https://bugzilla.redhat.com/show_bug.cgi?id=1463509#c13) and the conclusion in comment 13 was that we'd introduce these options but leave their defaults to match the current behaviour.

The modifications that the option refers to involve the creation of temporary namespaces and pods to help diagnose network connectivity, and they are expected to be "safe enough" and hence are enabled by default (i.e. `--prevent-modifications` is off by default).

Basically the new option is here to allow disabling when/if these diagnostics can cause a problem (e.g. if the cluster is running disconnected and doesn't have access to the images required to run these diagnostics).

Reply:
Thanks for confirming, I'll defer to that judgement in that case.